Makefile

```diff
@@ -20,15 +20,20 @@ build-cli:
 	./gradlew shadowJar
 
 javadoc:
+	rm -rf documentation
```

Comment:
somehow this scares me a little :) suggest add a `clean` option to Makefile, or adding `-p` to mkdir to not error if the directory exists

Reply:
actually, `documentation` is the temporary dir, which creates a transit place between dirs where docs are generated and publisher-production. It was useful for local tests. I'm gonna revert this line